src/aggregate-intro.rst

```diff
@@ -1,14 +1,37 @@
-******************
 ODK Aggregate
-******************
+===================
 
 .. _aggregate-introduction:
 
-ODK Aggregate is a server application and data repository used with :doc:`collect-intro` or other :doc:`OpenRosa-compliant data client applications <openrosa>` to:
+:dfn:`ODK Aggregate` is an open source Java application that
+stores, analyzes, and presents :doc:`XForm <form-design-intro>` survey data collected using :doc:`collect-intro` or other :doc:`OpenRosa-compliant applications <openrosa>`. It supports a wide range of data types, and is designed to work well in any hosting environment.
+
+With Aggregate, survey collection teams can:
```

Comment:
I usually say data collection. Survey collection reads funny to me.

Reply:
Funny weird or funny haha? :)